app/src/main/java/com/fluffybros/tujefe/ui/add/QRScanFragment.kt

```diff
@@ -0,0 +1,95 @@
+package com.fluffybros.tujefe.ui.add
+
+import android.Manifest
+import android.content.Intent
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.os.Environment
+import android.provider.MediaStore
+import android.util.Log
+import android.view.View
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.fluffybros.tujefe.R
+import com.fluffybros.tujefe.databinding.FragmentQrScanBinding
+import java.io.File
+import java.io.IOException
+import java.text.DateFormat.getDateTimeInstance
+
+class QRScanFragment : Fragment(R.layout.fragment_qr_scan) {
+    private val REQUEST_IMAGE_CAPTURE = 1
+    private lateinit var currentPhotoPath: String
```

Comment:
where is this used?

Reply:
In the createImageFile method. I also thought it wasn't really doing anything, but if I delete it, the program crashes when reading barcodes it works fine with otherwise.

Reply:
I assume the apply statement is not only modifying currentPhotoPath but also the File's absolutePath.